source/backend/api/Controllers/LookupController.cs

```diff
@@ -126,7 +126,6 @@ public IActionResult GetAll()
                 var dispositionInitiatingDocTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllDispositionInitiatingDocTypes());
                 var dispositionTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllDispositionTypes());
                 var dispositionOfferStatusTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllDispositionOfferStatusTypes());
-                var dispositionChecklistItemTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllDispositionChecklistItemTypes());
```

Comment:
question: I see this removed from here but the repository method is still there. Shouldn't we remove the code that fetches these lookups from the DB if we don't intend to expose them externally here?

I'm not sure if this deletion was intentional. Please clarify